workshop_app/templates/workshop_app/workshop_status.html

```diff
@@ -0,0 +1,190 @@
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
+{% endblock %}
+{% block extra-custom-scripts %}
+    <script>
+        //ToolTip popup function on-hover
+        $(document).ready(function () {
+            $('[data-toggle="popover"]').popover({
+                placement: 'top',
+                trigger: 'hover'
+            });
+        });
+
+        //ToolTip popup function on-hover
+        $(document).ready(function () {
+            $('[data-toggle="popinfo"]').popover({
+                placement: 'top',
+                trigger: 'hover'
+            });
+        });
+
+        // Change date modal
+        function changeDate(date) {
+            let previous_date = new Date(date);
+            let dateToday = new Date();
+            let upto = new Date();
+
+            previous_date.setDate(previous_date.getDate() + 1);
+            upto.setFullYear(dateToday.getFullYear() + 1);
+
+            let counter = date.split(" ");
+            const id = counter.slice(-1).pop();
+            if (date[0] === 'P') {
+                counter = '.pDate' + id
+                $(counter).datepicker({
+                    changeMonth: true,
+                    changeYear: true,
+                    minDate: dateToday,
+                    maxDate: upto,
+                    dateFormat: "yy-mm-dd",
+                });
+                $(".ui-dialog-content").dialog("close");
+                $('.myDialogP' + id).dialog();
+
+            } else {
+                counter = '.rDate' + id;
+                $(counter).datepicker({
+                    changeMonth: true,
+                    changeYear: true,
+                    minDate: dateToday,
+                    maxDate: upto,
+                    dateFormat: "yy-mm-dd",
+                });
+                $(".ui-dialog-content").dialog("close");
+                $('.myDialogR' + id).dialog();
+
+            }
+        };
+
+    </script>
+{% endblock %}
+
+
+{% block content %}
+    {% if workshops %}
+        <h3 class="text-center" style="color:#2952a2;">The status of your workshops </h3>
+        <br>
+
+        <!-- Accepted View -->
+        <br>
+        <div class="container">
+            <h3 align="center" style="color: #04a9cf;"><strong>Workshops Accepted</strong></h3>
+            <table class="table table-striped">
+                <thead>
+                <tr>
+                    <th>Coordinator Name</th>
+                    <th>Institute</th>
+                    <th>Workshop Name</th>
+                    <th>Workshop Day</th>
+                    <th>Status</th>
+                </tr>
+                </thead>
+                {% for workshop in workshops %}
+                    <tbody>
+                    <tr>
+                        {% if workshop.status %}
```

Comment:
Make sure that workshop status is a mandatory field in the DB, which means you will not need this condition here

Reply:
It is mandatory field. status=0 corrensponds to 'pending' and status=1 corresponds to accepted.
[(Defined here)](https://github.com/FOSSEE/workshop_booking/blob/83be359156350d851950f090dea9906d022e478b/workshop_app/models.py#L165)